omod/src/main/web/dashboardwidgets/obsgraph/obsgraph.controller.js

```diff
@@ -124,6 +124,10 @@ export default class ObsGraphController {
             if (xAxisKeys !== null && xAxisKeys.length > 0) {
               for (let j=0; j < this.conceptArray.length; j++) {
                 let concept = this.conceptArray[j];
+                if( this.hideConcepts !== undefined && this.hideConcepts.length && this.hideConcepts.includes(concept.uuid)) {
+                  //if it was indicated that this concept should not be plotted, than skip to the next concept
+                  continue;
```

Comment:
I'm probably missing something obvious :) but if you don't want to graph a concept, why would you include it in the concept list in the first place?

Reply:
So I could get the values I need to calculate the BMI. I need the Weight obs to calculate the BMI but we do not want to plot the Weight line.

Reply:
Ah, makes sense!